projects/angular/src/data/datagrid/API.md

```diff
@@ -0,0 +1,53 @@
+# Datagrid - Style API
+
+## CSS Variables
+
+| CSS Variable Name                                     | Description                                                                  |
+| ----------------------------------------------------- | -----------------------------------------------------------------------------|
+| --clr-datagrid-font-color                             | Changes the text color of datagrid.                                          |
+| --clr-datagrid-default-border-color                   | Changes the border color of datagrid when default.                           |
+| --clr-datagrid-icon-color                             | Changes the header icons color of datagrid.                                  |
+| --clr-datagrid-row-hover-color                        | Changes the row background color of the data table when hovers over element. |
```

Comment:
Changes the datagrid row color when hovered over.

Reply:
As per your suggestions, changed it.